Add jest test template (#4)
MetaMask/metamask-module-template: README.md

_Add examples here_

## Testing

> `npm test` or `yarn test` to run the tests once.
> 
> To watch, install [jest](https://jestjs.io/) globally, and run `jest --watch`, or `npm run watch` or `yarn watch`.

Reviewer: ```suggestion
Run `yarn test` to run the tests once.

To run tests on file changes, run `yarn test:watch`.
```
